<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenFolder">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtFolder">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSubfolders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subfolders</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblSubfolders">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExportJS">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export to Javascript</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExportFinImages">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Fin Images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowInMap">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in Map</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
